newtmgr/bll/bll_sesn_cfg_windows.go

```diff
@@ -32,8 +32,8 @@ type BllSesnCfg struct {
 	MgmtProto    sesn.MgmtProto
 	PreferredMtu int
 	ConnTimeout  time.Duration
-	TxFilterCb   nmcoap.MsgFilter
-	RxFilterCb   nmcoap.MsgFilter
+	TxFilter     nmcoap.TxMsgFilter
+	RxFilter     nmcoap.RxMsgFilter
```

Comment:
Does changing the variable names have no effect on the code using them?

Reply:
Maybe the commit message should explain why `MsgFilter` is not available on Windows?

Reply:
Updated commit message.

Reply:
Thanks, I should have suspected the reason! :-)